components/automate-ui/src/app/pages/+compliance/+reporting/reporting-searchbar/reporting-searchbar.component.ts

```diff
@@ -446,4 +458,30 @@ export class ReportingSearchbarComponent implements OnInit {
     this.keyInputVisible = false;
     this.valInputVisible = true;
   }
+
+  formatType(type: any) {
+    // format the type value when control tag key is selected.
+    return type.search('control_tag:') === 0 ? 'control_tag_value' : type;
+  }
+
+  // When value is clicked for the control tag key
+  onControlTagKeyValueClick(value: any) {
+    const reportQuery = this.reportQuery.getReportQuery();
+    const type = this.selectedType;
+    type.name = `control_tag:${value.text}`;
+    type.title = `Control Tag | ${value.text}`;
+    type.placeholder = 'Enter Control Tag Values...';
```

Comment:
Consider this slightly more encapsulated variation that shows the intent a bit more clearly:
```
const type = { ...this.selectedType,
  name: `control_tag:${value.text}`,
  title: `Control Tag | ${value.text}`,
  placeholder: 'Enter Control Tag Values...'};
```

Reply:
@msorens good suggestion 👍🏻